### PR TITLE
improve upgrade notes for 1.1.0

### PR DESCRIPTION
**Pull request by jolan**
Belated update of the upgrade guide/release notes.  Tried to clean things up and clarify the status of mainnet vs. testnet w/stakepoold as much as possible.

Let me know if anything was neglected or needs improvement.

@chappjc @girino @karamble @jyap808 @mverrilli @dyrk @Retord @walkeralencar @tpruvot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,85 +1,50 @@
...
+
+## 1.1.0 Testnet/stakepoold Guide (Work-In-Progress/Subject To Change)
+
+- Follow the mainnet directions. The stake version on testnet is 5 and there
+  is 1 agenda.
 - Enable MySQL access from the stakepoold hosts.
 - Configure stakepoold by editing stakepoold.conf.  You should place stakepoold
-  on the same server as dcrd/dcrwallet so stakepoold can talk to them via loopback
-  to lower latency as much as possible.
+  on the same server as dcrd/dcrwallet so stakepoold can talk to them via
```
Comment (chappjc) [inline README.md]: Where does the conf go?

**Comment by girino** [comment]
It would be nice to suggest securing mysql connections. something along the lines:

- Enable MySQL access from the stakepoold hosts, preferably using a VPN, SSH tunnel or other encrypted channel.

**Comment by jolan** [comment]
Well the stakepoold stuff is still subject to change.   I am more worried about the mainnet parts.

File: README.md (hunk 1 above)
Comment (chappjc) [inline README.md]: to clarify my comment, I assume it goes at `~/.stakepoold/stakepoold.conf`, but it would be helpful to make that explicit

**Pull request merged**